### ISSUE TITLE
false positive: Ignored files found that are tracked

### ISSUE THREAD

**Issue by KingMario** [hsz/idea-gitignore#361]
I have the following entries in .gitignore

```
frontend/node_modules/
!frontend/node_modules/clarity-angular/
!frontend/node_modules/clarity-ui/
```

But the plugin reports the following alarm 

> Ignored files found that are tracked: .ignore plugin found some files that are tracked with Git which are also listed in .gitignore file.

Click the _Show Detail_ link, and I find that it's about the files under `frontend/node_modules/clarity_angular` and  `frontend/node_modules/clarity-ui/`.

Besides, there are many _entry is never used_ alarms in the file which meaning is unsure.

![image](https://cloud.githubusercontent.com/assets/3956876/26038554/ab144726-393d-11e7-9c22-5803d2228313.png)

Did I understand something wrong?

**Comment by osahner** [comment]
I second that. Mostly gone after reopen.

**Comment by Hronom** [comment]
I'm using latest release of this plugin and also have problem with excluded folders:
```
*.jar
!/libs/**/*
```
The plugin wants untrack all `.jar` files inside `/libs` directory.

**Comment by mko-x** [comment]
Same Problem here with PHPStorm.

Project source code files are randomly marked as ignored with warning in IDE if opened. This happened several times to definitely not excluded or ignored files now.

![image](https://cloud.githubusercontent.com/assets/10264942/26399427/2e2b2be8-407d-11e7-95c2-6818a25991be.png)

**Comment by YOzaz** [comment]
Same here.

**Comment by hsz** [comment]
@KingMario @osahner @Hronom @mko-x @YOzaz @iandbanks

Thanks for this report!

This issue has been resolved - commit is attached to #350.
Fix will be released soon with v2.0.0 version.
